### PR TITLE
August 2022 TSC meeting minutes

### PR DESCRIPTION
**Pull request by weierophinney**
- Minutes from the meeting
- Agenda reset for September 2022 meeting

### CODE REVIEW

File: meetings/minutes/2022-08-01-TSC-Minutes.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+The impetus for this is two items:
+
+- We've observed rulesets change/rename in minor versions, which can cause breakage in our CI.
+- CodeSniffer itself has had breakage even in patch versions (though, as Luís notes, it's rare... but when it happens, it can be painful).
+
+Matthew W suggested that we probably should have a different rule for laminas-coding-standard with regard to new major releases, as any addition or rule change is essentially a BC break; as such, new major releases should be normalized for the tool.
+This was met with agreement.
+
+Max and Geert wondered if we should be pinning the _dependencies_ for laminas-coding-standard to minor versions as well, to prevent breakage.
```
Comment (internalsystemerror) [inline meetings/minutes/2022-08-01-TSC-Minutes.md:136]: @geerteltink suggested pinning to the patch version here.

File: meetings/minutes/2022-08-01-TSC-Minutes.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+#### Decision
+
+We decided to roll a new major, and start pinning dependencies to minor releases to prevent future headaches.
```
Comment (Xerkus) [inline meetings/minutes/2022-08-01-TSC-Minutes.md:140]: Same here. Decision was to pin to patch versions.

Comment (Ocramius) [reply meetings/minutes/2022-08-01-TSC-Minutes.md:140]: for PHPCS itself, not really for everything.

File: meetings/minutes/2022-08-01-TSC-Minutes.md (hunk 1 above)
Comment (weierophinney) [reply meetings/minutes/2022-08-01-TSC-Minutes.md:136]: Good catch - adding some verbiage around that.

File: meetings/minutes/2022-08-01-TSC-Minutes.md (hunk 2 above)
Comment (weierophinney) [reply meetings/minutes/2022-08-01-TSC-Minutes.md:141]: Clarifying that point in the minutes; thanks!

**Comment by weierophinney** [comment]
Updates made to indicate we are pinning phpcs to patch versions.

**Review by internalsystemerror — APPROVED** [review]

**Pull request merged**